models/transformation/dme_claims.sql

```diff
@@ -3,7 +3,7 @@ select
     , cast(clm_line_num as integer) as claim_line_number
     , 'professional' as claim_type
     , {{ cast_string_or_varchar('bene_mbi_id') }} as patient_id
-    , {{ cast_string_or_varchar('NULL') }} as member_id
+    , {{ cast_string_or_varchar('bene_mbi_id') }} as member_id

```

Comment:
I'm not sure this is necessary. We don't use member_id (as far as I know) in any downstream logic. I believe this is just here so that someone can link a member to different LOBs when doing ad-hoc analytics. I believe the SAF data is all one data product.

Reply:
My opinion is we should populate member id across the board, and if we don't have a separate concept, it should just be duplicative of patient.  I think that's what we should suggest to users who have a patient concept but not a member concept. Some of our marts probably need to be updated to include member as well as patient (e.g. pMpm).

I think it makes the most sense from a data perspective.  If you're looking for a unique id at member grain but you don't have a separate member concept, the each unique patient is also a unique member.

Reply:
leaving as is for now.  I think we should reflect on both how member_id and data_source are used in all of ourpackages at some point, I dont think many of our packages would work as designed today if there were patients with multiple members or data sources with patient ids that were duplicated across sources